pkg/scheduler/plugins/elasticquota/plugin_helper.go

```diff
@@ -64,16 +64,17 @@ func (g *Plugin) migrateDefaultQuotaGroupsPod() {
 	for _, pod := range defaultQuotaInfo.GetPodCache() {
 		quotaName := g.getPodAssociateQuotaName(pod)
 		if quotaName != extension.DefaultQuotaName {
-			isAssigned := g.groupQuotaManager.GetPodIsAssigned(extension.DefaultQuotaName, pod)
-			g.groupQuotaManager.UpdatePodRequest(extension.DefaultQuotaName, pod, nil)
-			g.groupQuotaManager.UpdatePodUsed(extension.DefaultQuotaName, pod, nil)
-			g.groupQuotaManager.UpdatePodCache(extension.DefaultQuotaName, pod, false)
-
-			g.groupQuotaManager.UpdatePodCache(quotaName, pod, true)
-			g.groupQuotaManager.UpdatePodIsAssigned(quotaName, pod, isAssigned)
-			g.groupQuotaManager.UpdatePodRequest(quotaName, nil, pod)
-			g.groupQuotaManager.UpdatePodUsed(quotaName, nil, pod)
-			klog.V(5).Infof("migrate pod :%v from defaultQuotaGroup to quota:%v, podPhase:%v", pod.Name, quotaName, pod.Status.Phase)
+			g.groupQuotaManager.MigratePodFromLeftToRight(pod, extension.DefaultQuotaName, quotaName)
+		}
+	}
+}
+
+// migratePodsToDefaultQuotaGroup if a quotaGroup is deleted, migrate its pods to defaultQuotaGroup
+func (g *Plugin) migratePodsToDefaultQuotaGroup(quotaName string) {
```

Comment:
migratePodsToDefaultQuotaGroup-> migratePods(out, in string),  pass extension.DefaultQuotaName as a parameter.

Reply:
fixed